plugin-server/src/worker/ingestion/app-metrics.ts

```diff
@@ -53,6 +53,21 @@ interface QueuedMetric {
     metric: AppMetricIdentifier
 }
 
+/** An aggregated AppMetric, as written to/read from a ClickHouse row. */
+export interface RawAppMetric {
```

Comment:
I saw some other cases of `Raw`- prefix in the codebase for this type of intermediate representation, so just followed that pattern. `ClickHouseAppMetric` or `AppMetricRow` or something like that would also work.